src/gui/shellwidget/shellwidget.cpp

```diff
@@ -570,6 +566,8 @@ void ShellWidget::paintEvent(QPaintEvent *ev)
 {
 	QPainter p(this);
 
+	p.setRenderHint(QPainter::Antialiasing);
```

Comment:
I don't think this is valid as-written. I think this is the cause of ghosting in my screenshot?